torch_geometric/datasets/linkx_dataset.py

```diff
@@ -32,23 +32,45 @@ class LINKXDataset(InMemoryDataset):
             being saved to disk. (default: :obj:`None`)
     """
 
-    url = 'https://github.com/CUAI/Non-Homophily-Large-Scale/raw/master/data'
+    github_url = 'https://github.com/CUAI/Non-Homophily-Large-Scale/' \
+                 'raw/master/data'
+    gdrive_url = 'https://drive.google.com/uc?confirm=t&'
 
     facebook_datasets = [
         'penn94', 'reed98', 'amherst41', 'cornell5', 'johnshopkins55'
     ]
 
     datasets = {
-        'penn94': f'{url}/facebook100/Penn94.mat',
-        'reed98': f'{url}/facebook100/Reed98.mat',
-        'amherst41': f'{url}/facebook100/Amherst41.mat',
-        'cornell5': f'{url}/facebook100/Cornell5.mat',
-        'johnshopkins55': f'{url}/facebook100/Johns%20Hopkins55.mat',
-        'genius': f'{url}/genius.mat'
+        'penn94': {
```

Comment:
This is maybe not the cleanest way to handle this - but I wanted to find a good way to address the need for multiple files in the wiki dataset, while leaving the rest of the dataset abstractions in place. 

Feel free to suggest an alternative anyone.